src/operator.rs

```diff
@@ -149,27 +149,27 @@ impl<'a, 'mir, 'tcx> EvalContextExt<'tcx> for super::MiriEvalContext<'a, 'mir, '
                 if left.alloc_id == right.alloc_id {
                     left.offset == right.offset
                 } else {
-                    // This accepts one-past-the end.  So technically there is still
+                    // This accepts one-past-the end. Thus, there is still technically
```

Comment:
The comma seems weird. Also the technically makes more sense to me before the "there".

Reply:
The "technically" sounds equally natural either before or after, in honesty. The comma is necessary.